actor/builtin/miner/miner.go

```diff
@@ -463,9 +463,11 @@ func (ma *Actor) CommitSector(ctx exec.VMContext, sectorID uint64, commD, commR,
 		// It is undefined behavior for a miner using "LiveMode" to verify a
 		// proof created by a miner using "TestMode" (and vice-versa).
 		//
-		proofsMode := proofs.LiveMode
+		var sectorSize types.SectorSize
 		if os.Getenv("FIL_USE_SMALL_SECTORS") == "true" {
-			proofsMode = proofs.TestMode
+			sectorSize = types.OneKiBSectorSize
+		} else {
```

Comment:
non-blocking: Does this conditional go away once the miner has sector size in state?

Reply:
Yep. The proofs-mode switching will happen only in the `StorageMarketActor#CreateMiner` method (to ensure that the selected sector size is a valid option for the currently-configured mode). Also, the switching will happen on a property _of_ the `StorageMarketActor` instead of an environment variable.